packages/teuchos/core/src/CMakeLists.txt

```diff
@@ -32,6 +32,7 @@ SET(HAVE_TEUCHOSCORE_CXX11 ON)
 SET(YAML_CPP_DEF 0)
 IF(${PACKAGE_NAME}_ENABLE_yaml-cpp)
  SET(YAML_CPP_DEF 1)
+ SET(HAVE_TEUCHOSCORE_YAMLCPP ON)
 ENDIF()
```

Comment:
This yaml block can be moved into the top level CMakeLists.txt file in the TeuchosParameterList subpackage.

Reply:
Fixed